content/js/query.md

```diff
@@ -67,30 +67,33 @@ skygear.publicDB.query(query).then((notes) => {
 Here is a list of simple conditions you can apply on a query:
 
 ``` javascript
-query.equalTo('title', 'First note');      // ===
-query.notEqualTo('category', 'dangerous'); // !==
-query.lessThan('age', 10);                 // <
-query.lessThanOrEqualTo('age', 10);        // <=
-query.greaterThan('size', 50);             // >
-query.greaterThanOrEqualTo('size', 50);    // >=
+query.equalTo('title', 'First note');
+query.notEqualTo('category', 'dangerous');
+query.lessThan('age', 10);
+query.lessThanOrEqualTo('age', 10);
+query.greaterThan('size', 50);
+query.greaterThanOrEqualTo('size', 50);
 ```
 
 ### Contains condition
 
 The `contains` condition can be used to query a key for value that matches one
-of the items in a specified array.
+of the items in a specified array. Suppose here we are querying for movies,
+and each movie has exactly one genre.
 
-```javascript
-query.contains('category', ['interesting', 'funny']);
-query.notContains(...);
+``` javascript
+query.contains('genre', ['science-fiction', 'adventure']);
+query.notContains('genre', ['romance', 'horror']);
 ```
 
 If the value is an array, the `containsValue` condition can be used to query for
-a key that has an array as its value containing the specified value.
+a key that has an array as its value containing the specified value. Suppose
+further that each movie has multiple tags as an array, for example
+_Independence Day: Resurgence_ may have tags extraterrestrial and war.
 
-```javascript
-query.containsValue('category', 'interesting');
-query.notContainsValue(...);
+``` javascript
+query.containsValue('tags', 'hollywood');
+query.notContainsValue('tags', 'war');
```

Comment:
do you think reversing `hollywood` and `war` will be clearer since both would match the example movie mentioned above?